src/utils/interface.js

```diff
@@ -1,3 +1,6 @@
+
+const KEY_CODES = { q: 81, v: 86 };
```

Comment:
Those are the ASCII codes for the capital letters "Q" and "V", so holding shift almost certainly doesn't change what code is sent.